src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/.template.config.src/dotnetcli.host.json

```diff
@@ -11,7 +11,7 @@
     }
   },
   "usageExamples": [
-    "--framework net7.0",
-    "--orchard-version 1.6.0"
+    "--framework net8.0",
+    "--orchard-version 1.8.3"
```

Comment:
Please add that we need to change these as well to https://github.com/OrchardCMS/OrchardCore/blob/main/.github/ISSUE_TEMPLATE/release.md, please. These don't show up in a default File types = `!*\bin\*;!*\obj\*;!*\.*;` VS search, not even when explicitly searching for `*.json` files:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/5c61b817-6add-4725-a3aa-7251a3a2947e)